gsoc/gsoc2016/ideas_eras.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: default
+title:  "Ideas for IMS - ERAS"
+show_main: false
+ideas_team: IMS
+---
+
+### 1 – SOLAR STORMS FORECASTING SERVER ###
+
+*Suggested Mentors:* [Antonio del Mastro](https://bitbucket.org/aldebran/) , [Olena Persianova](https://bitbucket.org/OPersian/)
```

Comment:
@mtambos I'm merging it now so students can see the project offered. Please, update this with a new pull request later.